tests/org.jboss.tools.cdk.ui.bot.test/pom.xml

```diff
@@ -80,8 +80,27 @@
 					<testClass>${suiteClass}</testClass>
 				</configuration>
 			</plugin>
+            <plugin>
+				<groupId>org.eclipse.tycho</groupId>
+				<artifactId>target-platform-configuration</artifactId>
+				<configuration>
+                      <dependencies combine.children="append">
+                          <dependency>
+                              <type>p2-installable-unit</type>
+                              <artifactId>org.eclipse.launchbar.feature.group</artifactId>
+                              <version>0.0.0</version>
+                          </dependency>
+                      </dependencies>
+				</configuration>
+			</plugin>
 		</plugins>
```

Comment:
This part here servers as a workaround for https://issues.jboss.org/browse/JBIDE-24878. Removal of this part is considered in https://issues.jboss.org/browse/JBIDE-24970.